packages/client-core/src/admin/components/Project/ProjectTable.tsx

```diff
@@ -78,11 +80,15 @@ const ProjectTable = ({ className }: Props) => {
   const [rowsPerPage, setRowsPerPage] = useState(PROJECT_PAGE_LIMIT)
   const [changeDestination, setChangeDestination] = useState(false)
 
-  const projectState = useHookstate(getMutableState(ProjectState))
-  const adminProjects = projectState.projects
-  const adminProjectCount = adminProjects.value.length
+  const projectsQuery = useFind('project', { query: { allowed: true } })
```

Comment:
This will only return us 10 records. We should handle pagination: false for this endpoint.

Reply:
implemented!